lib/ddtrace/span.rb

```diff
@@ -65,8 +65,8 @@ def get_tag(key)
       @meta[key]
     end
 
-    # Set the given key / value metric pair on the span. Keys must be string.
-    # Values must be floating point numbers.
+    # This method sets a tag for a floating point value for the given key. It acts
```

Comment:
yes, we don't advocate `set_metric()` at all, indeed it's not in the docs and there is not a particular reason to use it instead of `set_tag()`. Here I just want to write something that is explicit that `set_metric()` acts like `set_tag()` and does not have anything to do with infrastructure metrics.

Do you think we can reword it better?